Resources/Models/plane.obj

```diff
@@ -0,0 +1,14 @@
+# Blender v2.82 (sub 7) OBJ File: ''
+# www.blender.org
+v -1.000000 0.000000 1.000000
+v 1.000000 0.000000 1.000000
+v -1.000000 0.000000 -1.000000
```

Comment:
what are these files? should a README.md be added to either `Resources` or `Resources/Models` explaining what's in here?

Reply:
I added a read me file in the Resource folder